privileged_residues/util.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+import pyrosetta
+
+from pyrosetta.bindings.utility import bind_method
+from pyrosetta.rosetta.core.import_pose import pose_from_pdbstring
+from pyrosetta.rosetta.numeric import xyzVector_double_t as V3
+from pyrosetta.toolbox.numpy_utils import numpy_to_rosetta
+
+from rif.geom import Ray
+
+@bind_method(pyrosetta.Pose)
+def apply_transform(self, xform):
+    """Apply a homogeneous transform to the current pose.
+
+    Parameters
+    ----------
+    xform : np.ndarray
+        A homogeneous transform.
+    """
+
+    assert(xform.shape == (4, 4)) # homogeneous transform
```

Comment:
There's a better assertion here. I think rotations have determinant == 1. so let's check Rx for that.